plugins/gfm/src/main/kotlin/org/jetbrains/dokka/gfm/gfmTemplating.kt

```diff
@@ -9,11 +9,20 @@ import com.fasterxml.jackson.annotation.JsonTypeInfo.Id.CLASS
 sealed class GfmCommand {
     companion object {
         private const val delimiter = "\u1680"
-        fun templateCommand(command: GfmCommand): String = "$delimiter GfmCommand ${toJsonString(command)}$delimiter"
-        val templateCommandRegex: Regex = Regex("$delimiter GfmCommand ([^$delimiter ]*)$delimiter")
+        val templateCommandRegex: Regex =
+            Regex("<!---$delimiter GfmCommand ([^$delimiter ]*)$delimiter--->(.+?)(?=<!---$delimiter)<!---$delimiter--->")
+        val MatchResult.command
+            get() = groupValues[1]
+        val MatchResult.label
+            get() = groupValues[2]
+        fun Appendable.templateCommand(command: GfmCommand, content: Appendable.() -> Unit): Unit {
+            append("<!---$delimiter GfmCommand ${toJsonString(command)}$delimiter--->")
```

Comment:
since this is a multiline command will it be ok when the content is inside a table? Markdown table cells can't have multiline contents